edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -213,9 +224,18 @@ def _get_missing_cookie_message_and_attribute(self, cookie_name):
     def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: disable=unused-argument
         """
         Reconstitute the full JWT and add a new cookie on the request object.
+
+        Additionally, may add the user to the request to make it available in process_view. (See below.)
         """
         assert hasattr(request, 'session'), "The Django authentication middleware requires session middleware to be installed. Edit your MIDDLEWARE setting to insert 'django.contrib.sessions.middleware.SessionMiddleware'."  # noqa E501 line too long
 
+        is_forgiving_jwt_cookies_enabled = get_setting(ENABLE_FORGIVING_JWT_COOKIES)
+        if is_forgiving_jwt_cookies_enabled:
+            self._process_view_forgiving_jwt_cookies(request, view_func)
+        else:
+            self._process_view_original(request, view_func)
```

Comment:
Suggested recombination of the two methods: https://github.com/openedx/edx-drf-extensions/pull/366

Reply:
Done.